label_studio/projects/models.py

```diff
@@ -916,3 +924,8 @@ def remove_created_annotations_and_labels(self, annotations):
         self.created_annotations = created_annotations
         self.created_labels = labels
         self.save()
+
+
+Q_finished_annotations = Q(annotations__was_cancelled=False) & \
```

Comment:
We already have `Q_finished_annotations` in `tasks.models`. I think we can add `Q(annotations__ground_truth=False)` to it, fix the only place where it is used, and import it there without defining a new one.

Reply:
Fixed by using Q from tasks.models